commands/databases.go

```diff
@@ -909,3 +910,54 @@ func displayDatabaseReplicaConnection(c *CmdConfig, conn do.DatabaseConnection)
 	item := &displayers.DatabaseConnection{DatabaseConnection: conn}
 	return c.Display(item)
 }
+
+func sqlMode() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:     "sql-mode",
+			Aliases: []string{"sm"},
+			Short:   "database sql-mode commands",
+			Long:    "sql-mode is used to access database sql-mode commands",
+		},
+	}
+
+	CmdBuilder(cmd, RunDatabaseGetSQLModes, "get <database-id>",
```

Comment:
get should have a `g` alias for the command.

Reply:
will do